protocol/protocol.proto

```diff
@@ -805,6 +825,10 @@ service StreamService {
     rpc AddEvent(AddEventRequest) returns (AddEventResponse);
     rpc SyncStreams(SyncStreamsRequest) returns (stream SyncStreamsResponse);
     rpc AddStreamToSync(AddStreamToSyncRequest) returns (AddStreamToSyncResponse);
+    // AddStreamsToSyncRequest adds multiple streams to an in progress sync operation.
+    // The client must check AddStreamsToSyncResponse.errors to determine if some streams failed to add.
+    // Note that it is possible that the client receives stream updates before the calls returned.
+    rpc AddStreamsToSync(AddStreamsToSyncRequest) returns (AddStreamsToSyncResponse);
```

Comment:
Lets do `ModifySync` which also can remove streams (this way it will cover all use cases for batch updates).